zookeeper/cli.py

```diff
@@ -61,6 +61,11 @@ def build_train(function):
         default=False,
         help="If you want to split a dataset which only contains a train/test into train/val/test",
     )
+    @click.option(
+        "--autotune/--no-autotune",
+        default=True,
+        help="Use tf.data.experimental.AUTOTUNE for data pipeline.",
+    )
```

Comment:
Please add a usage example to the docs explaining why one may not want to autotune